install/helm/agones/templates/crds/_gameserverspecschema.yaml

```diff
@@ -153,6 +153,42 @@ properties:
             type: integer
             title: The initial player capacity of this Game Server
             minimum: 0
+{{- if .featureCountsAndLists }}
+      counts:
+        type: array
+        title: Array of player, room, session, etc. counters
+        nullable: true
+        maxItems: 1000
+        items:
+          type: object
+          properties:
+            name: # counter name
+              type: string
+            count: # current count
```

Comment:
should it have a `minimum: 0`?

Reply:
Yes, planning on doing that as part of the validation PR (coming up next :))